go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/godoctor/godoctor
+
+go 1.12
```

Comment:
I suppose we could make this a lower number since it's not strictly required, though it's possible we won't really scare anyone off with this one (if somebody squeaks, we can fuss with it)